.github/workflows/test-code.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         working-directory: src
         run: |
           dotnet tool install -g dotnet-format --version 6.0.* --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
+          dotnet format --check --fix-whitespace || (echo "Run 'dotnet format' to fix formatting issues" && exit 1)
```

Comment:
`-w, --fix-whitespace                                                       Run whitespace formatting. Run by default when not applying fixes.` 🤦‍♂️